src/workloads/query/multiplanner/Simple.yml

```diff
@@ -0,0 +1,277 @@
+        # It has to be a separate actor because they target different DBs.
+      active: [0]
+      nopInPhasesUpTo: *maxPhase
+      collection: *coll
+
+- Name: CreateDataset
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: InsertTemplate
+    Parameters:
+      database: *db
+      threadsA: 1
+      threadsB: 1
+      active: [1]
+      nopInPhasesUpTo: *maxPhase
+      repeat: 1
+      collection: *coll
+      docCount: *docCount
+      indexes:
+        - keys: {x1: 1}
+        - keys: {x2: 1}
+        - keys: {x3: 1}
+        - keys: {x4: 1}
+        - keys: {x5: 1}
+        - keys: {x6: 1}
+        - keys: {x7: 1}
+        - keys: {x8: 1}
+        - keys: {x9: 1}
+        - keys: {x10: 1}
+        - keys: {x11: 1}
+        - keys: {x12: 1}
+        - keys: {x13: 1}
+        - keys: {x14: 1}
+        - keys: {x15: 1}
+        - keys: {x16: 1}
+        - keys: {x17: 1}
+        - keys: {x18: 1}
+        - keys: {x19: 1}
+        - keys: {x20: 1}
+        - keys: {x21: 1}
+        - keys: {x22: 1}
+        - keys: {x23: 1}
+        - keys: {x24: 1}
+        - keys: {x25: 1}
+        - keys: {x26: 1}
+        - keys: {x27: 1}
+        - keys: {x28: 1}
+        - keys: {x29: 1}
+        - keys: {x30: 1}
+        - keys: {x31: 1}
+        - keys: {x32: 1}
+        - keys: {x33: 1}
+        - keys: {x34: 1}
+        - keys: {x35: 1}
+        - keys: {x36: 1}
+        - keys: {x37: 1}
+        - keys: {x38: 1}
+        - keys: {x39: 1}
+        - keys: {x40: 1}
+        - keys: {x41: 1}
+        - keys: {x42: 1}
+        - keys: {x43: 1}
+        - keys: {x44: 1}
+        - keys: {x45: 1}
+        - keys: {x46: 1}
+        - keys: {x47: 1}
+        - keys: {x48: 1}
+        - keys: {x49: 1}
+        - keys: {x50: 1}
+        - keys: {x51: 1}
+        - keys: {x52: 1}
+        - keys: {x53: 1}
+        - keys: {x54: 1}
+        - keys: {x55: 1}
+        - keys: {x56: 1}
+        - keys: {x57: 1}
+        - keys: {x58: 1}
+        - keys: {x59: 1}
+        - keys: {x60: 1}
+        - keys: {x61: 1}
+        - keys: {x62: 1}
+        - keys: {x63: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: SimpleMultiplannerWith64Indexes
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [3]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      selectivity: *selectivity
+  
+- Name: Hide48Indexes
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Operations:
+        - {OperationName: RunCommand, OperationCommand: {collMod: *coll, index: {hidden: true, keyPattern: {x17: 1}}}}
```

Comment:
I'm not sure this is possible. I don't believe I can use `^FormatString` in conjunction with `^Inc` to succinctly name all the indexes. Is this similar to what you had in mind?
`- {OperationName: RunCommand, OperationCommand: {collMod: *coll, index: {hidden: true, keyPattern: {{^FormatString: {"{x%d_1", "withArgs:": {^Inc: {start: 17}}}}}}}}`

Reply:
For example, imagine trying to factor it out like this:

```
place_for_macro_definitions:
  my_macro:
    OperationName: RunCommand
    OperationCommand:
      collMod: *coll
      index: 
        hidden: true
        keyPattern: {^Parameter: {Name: "keyPattern", Default: "invalid"}}
...
Operations:
- {invoke_a_macro: 'my_macro', args: {keyPattern: {x1: 1}}}
- {invoke_a_macro: 'my_macro', args: {keyPattern: {x2: 1}}}
...
- {invoke_a_macro: 'my_macro', args: {keyPattern: {x63: 1}}}
```

There would still be repetition, which we need to verify by eye, but it also adds an indirection that can make it less clear what an individual command looks like.

Reply:
That makes sense to me, but is that preferred stylistically? It would be more lines of code and less readable than just repeating the same 64 lines where it is very clear that the same operation is being done to each index.

Reply:
Right, to be clear I was arguing that it wouldn't be an improvement overall to factor it out that way.

Reply:
That's an interesting approach, and I agree with @dpercy that it is better to be explicit. I was thinking of something similar to [this](https://github.com/mongodb/genny/pull/1134#discussion_r1513153261), but it is not important. Feel free to ignore this suggestion, better to be explicit.